internal/events/webhooks/webhooks.go

```diff
@@ -403,7 +402,7 @@ func (wh *WebHooks) attemptRequest(ctx context.Context, sub *core.Subscription,
 		Status:  resp.StatusCode(),
 		Headers: fftypes.JSONObject{},
 	}
-	// log.L(wh.ctx).Infof("Webhook<- %s %s event %s on subscription %s returned %d", req.method, req.url, event.ID, sub.ID, res.Status)
+	log.L(wh.ctx).Infof("Webhook<- %s %s on subscription %s returned %d", req.method, req.url, sub.ID, res.Status)
```

Comment:
Should this be `INFO` or `DEBUG` level? Looks like the corresponding log message above is `DEBUG`.

Reply:
Sure happy to change it to DEBUG